src/BigData/services/AbstractBigDataEntries.jl

```diff
@@ -125,24 +124,25 @@ Related
 
 addDataEntry!, addData!, fetchData, fetchDataEntryElement
 """
-function getDataEntryElement(dfg::AbstractDFG,
-                             dfglabel::Symbol,
-                             datastore::Union{FileDataStore, InMemoryDataStore},
-                             datalabel::Symbol)
+function getDataEntryBlob(dfg::AbstractDFG,
+                          dfglabel::Symbol,
+                          datastore::Union{FileDataStore, InMemoryDataStore},
+                          datalabel::Symbol)
   #
   vari = getVariable(dfg, dfglabel)
   if !hasDataEntry(vari, datalabel)
-    @error "missing data entry $datalabel in $dfglabel"
-    return nothing, nothing
+      # current standards is to fail hard
+    error("missing data entry $datalabel in $dfglabel")
```

Comment:
Gotcha, so fail loud :+1:

Reply:
Jip, as per #145. I would still like the option of not failing hard though. We can first standardize the behaviour and then add to it.